src/ralph/admin/templates/admin/templatetags/transition_history.html

```diff
@@ -0,0 +1,44 @@
+{% load i18n %}
+<div class="row">
+  <div class="lerge-12 columns">
+    <fieldset>
+      <legend>{% trans "Transition history" %}</legend>
+      {% if transitions_history %}
+      <table>
+        <thead>
+        <tr>
+          <th>{% trans "Date" %}</th>
+          <th>{% trans "Author" %}</th>
+          <th>{% trans "Action" %}</th>
+          <th>{% trans "Data" %}</th>
+          <th>{% trans "Attachment" %}</th>
+        </tr>
+        </thead>
+        <tbody>
+          {% for history in transitions_history %}
+            <tr>
+              <td>{{ history.date }}</td>
+              <td>{{ history.user }}</td>
+              <td>{{ history.action }}</td>
+              <td>
+                {% for value in history.data %}
+                  {{ value.0 }}: {{ value.1 }}<br />
+                {% endfor %}
+              </td>
+              <td>
+                  {% if history.attachment %}
+                    <a href="{% url history.attachment_url_name history.attachment.id history.attachment.original_filename %}">
+                      {{ history.attachment.original_filename }}
+                    </a>
+                  {% endif %}
```

Comment:
add ``{% else %}`` statement with ``<empty>`` or something like that for better UX - what's means blank field? Error?